wurst.build

```diff
@@ -1,2 +1,3 @@
 ---
 projectName: Wurst Standard Library 2
+dependencies: []
```

Comment:
Drive-by fix: without this line, `grill install` creates a copy of stdlib in `_build`